include/odp/api/spec/packet_io.h

```diff
@@ -489,6 +489,14 @@ typedef union odp_pktout_config_opt_t {
 		 */
 		uint64_t no_packet_refs  : 1;
 
+		/** Enable packet transmit completion event requests
+		 *
+		 * Use pktio capability tx_compl to check if TX completion events are supported.
+		 *
+		 * 0: Application will not request packet TX completion events (default)
+		 * 1: Application will request packet TX completion events
```

Comment:
Similar to the comment in the other PR, this could be "...may request..." instead of "...will request...".